packages/presentation/src/icons/iconsByName.ts

```diff
@@ -99,6 +99,7 @@ import { ReactComponent as mdUnknown } from './vectors/mdUnknown.svg';
 import { ReactComponent as mdConstraintGeneric } from './vectors/mdConstraintGeneric.svg';
 import { ReactComponent as mdConstraintDependsOn } from './vectors/mdConstraintDependsOn.svg';
 import { ReactComponent as mdConstraintAllowedTimes } from './vectors/mdConstraintAllowedTimes.svg';
+import { ReactComponent as mdVerification } from './vectors/mdVerification.svg';
```

Comment:
don't forget to remove this stuff in `/packages/presentation` if it's already been committed in the other PR